c2corg_ui/templates/outing/helpers/view.html

```diff
@@ -204,10 +204,10 @@
         <p><b class="value-title" translate>elevation_access</b>:  <span class="value">${outing['elevation_access']}&nbsp;m</span></p>
         % endif
 
-       ${get_document_up_down(outing, 'height_diff')}
+        ${get_document_up_down(outing, 'height_diff')}
 
         % if outing.get('length_total'):
-        <p><span class="value-title" translate>length_total</span>: <span class="value">${outing['length_total']}&nbsp;m</span></p>
+        <p><span class="value-title" translate>length_total</span>: <span class="value">${outing['length_total']/1000}&nbsp;km</span></p>
```

Comment:
The data is stored in meters. With this change, this attribute will for instance be displayed as follows:
* 1000 m => 1.0 km
* 12345 m => 12.345 km
* 12400 m => 12.4 km
* 150 m => 0.15 km

Actually you can already test in prod/demo for the route's ``route_length`` since that attribute was already displayed like this in master:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/route/helpers/view.html#L170